### DISCUSSION TITLE
1. October 2024

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#2006] [Meeting notes]
Agenda:
- PRs:
   - Go over PRs -- which ones can be closed?
   - https://github.com/gramineproject/gramine/pull/1795 -- discuss pros and cons
   - https://github.com/gramineproject/gramine/pull/1681 -- discuss how to proceed
   - https://github.com/orgs/gramineproject/projects/1 -- fix Roadmap
- Encrypted files recovery feature, as discussed in https://github.com/gramineproject/gramine/discussions/1972

**Comment by kailun-qin** [comment]
> Go over PRs -- which ones can be closed?
> https://github.com/gramineproject/gramine/pull/1681 -- discuss how to proceed

Since we're here (in discussing encrypted files), can we move https://github.com/gramineproject/gramine/pull/1845 forward (maybe after the release)?
It's been presented in previous meetings and it's in a good shape IMO. It should be helpful to Gramine users and/or developers.

Maybe also "encrypted files recovery feature", as discussed in https://github.com/gramineproject/gramine/discussions/1972 -- mostly want to hear @mkow's opinions.
